crates/storage/nippy-jar/src/lib.rs

```diff
@@ -85,7 +86,7 @@ pub struct NippyJar<H = ()> {
     /// Number of data columns in the jar.
     columns: usize,
     /// Optional compression algorithm applied to the data.
-    compressor: Option<Compressors>,
+    pub compressor: Option<Compressors>,
```

Comment:
let's not mix pub and priv fields but rather add a getter if we need access to this